File: frontend/src/Types/dailyMeetingEntry.ts
import type { Guid } from './api';

export interface DailyMeetingEntry {
  id: Guid;
  meetingId: Guid;
  personId: Guid;
  yesterday: string;
  today: string;
  blockers: string | null;
}

export interface CreateDailyMeetingEntryRequest {
  meetingId: Guid;
  personId: Guid;
  yesterday: string;
  today: string;
  blockers?: string | null;
}

export interface UpdateDailyMeetingEntryRequest {
  meetingId: Guid;
  personId: Guid;
  yesterday: string;
  today: string;
  blockers?: string | null;
}

File: backend/src/ScrumLog.Api/Data/Entities/DailyMeetingEntry.cs
namespace ScrumLog.Api.Data.Entities;

public class DailyMeetingEntry
{
    public Guid Id { get; set; }

    public Guid MeetingId { get; set; }

    public Meeting Meeting { get; set; } = null!;

    public Guid PersonId { get; set; }

    public Person Person { get; set; } = null!;

    public required string Yesterday { get; set; }

    public required string Today { get; set; }

    public string? Blockers { get; set; }
}
